salt/renderers/gpg.py

```diff
@@ -111,17 +111,23 @@
 DEFAULT_GPG_KEYDIR = os.path.join(salt.syspaths.CONFIG_DIR, 'gpgkeys')
 
 
-def decrypt_ciphertext(cypher, gpg):
+def decrypt_ciphertext(cypher, gpg, safe=False):
```

Comment:
Hm, but this argument is never used anywhere? It seems that it is now just hardcoded to `False` so this gives the same behaviour as before, where decryption failures are just ignored? So this doesn't really fix anything or am I missing something here?